DOCS-10267 Asana Connector 1.0.0 (#1322)
mulesoft/docs-connectors: asana/1.0/modules/ROOT/pages/index.adoc

= Asana Connector 1.0 - Mule 4

Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]

Anypoint Connector for Asana (Asana Connector) synchronizes data and automates business processes between Asana and third-party applications, either on-premises or in the cloud.

> Asana is a SaaS project management tool focused at enhancing team collaboration. Using Asana, employees can easily manage projects and tasks through one integrated collaborative project management interface.

Author: ```suggestion
Asana is a SaaS project management tool that enhances team collaboration. Employees can easily manage projects and tasks through one integrated collaborative project management interface when using Asana.
```

Reviewer: Fixed.

For compatibility information and fixed issues, see the Asana Connector Release Notes.

== Prerequisites

To use this connector, you must be familiar with:

* The Asana APIs
* Anypoint Connectors
* Mule runtime engine (Mule)
* Elements and global elements in a Mule flow
* Creating a Mule app using Anypoint Studio (Studio)

Before creating an app, you must have:

* Credentials to access the target resource
* Anypoint Studio version 7.3.5 or later

== Common Use Cases for the Connector

> Automate powerful business processes by integrating Asana with other IT tools and systems in your enterprise such as messaging, calendar, and issue triaging system. Possibilities include:

Author: ```suggestion
Automate powerful business processes by integrating Asana with other IT tools and systems in your enterprise such as messaging, calendar, and issue triaging systems. Possible uses include:
```

Reviewer: Fixed.
